cfgov/tccp/serializers.py

```diff
@@ -6,8 +6,14 @@
 
 class CardSurveyDataSerializer(serializers.HyperlinkedModelSerializer):
     purchase_apr_good_rating = serializers.IntegerField()
+    purchase_apr_good_min = serializers.FloatField()
+    purchase_apr_good_max = serializers.FloatField()
     purchase_apr_great_rating = serializers.IntegerField()
```

Comment:
Would you mind reordering all of these fields to be alphabetical, to match how it's done in CardSurveyDataListSerializer below?

Reply:
@chosak, fixed in b91b1f8